.github/workflows/linters.yml

```diff
@@ -6,18 +6,6 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  lighthouse:
-    name: Lighthouse
-    runs-on: ubuntu-22.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Lighthouse
-        run: npm install -g @lhci/cli@0.7.x
-      - name: Lighthouse Report
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```

Comment:
- [ ] You're not meant to alter any sections of your linters file. Kindly return the deleted section you removed 👍